.gitignore

```diff
@@ -19,3 +19,4 @@ testpullfilecache*
 package-lock.json
 yarn.lock
 /.vs
+typings/types.d.ts
```

Comment:
I added this to ignore conflicts in our PRs

Reply:
Should `types.d.ts` be deleted?

Reply:
Probably not, to accidentally not to release a package without types.

Reply:
And types will stay outdated in code

Reply:
Ah, you mean remove from git? Ok